recipe/meta.yaml

```diff
@@ -7,14 +7,19 @@ package:
 source:
   # We use the pypi URL as it includes the prepared Cython files.
   url: https://pypi.io/packages/source/p/pyzmq/pyzmq-{{ version }}.tar.gz
-  sha256: 824ad5888331aadeac772bce27e1c2fbcab82fade92edbd234542c4e12f0dca9
+  sha256: 6d18c76676771fd891ca8e0e68da0bbfb88e30129835c0ade748016adb3b6242
 
 build:
   number: 1
+  rpaths:
+    - lib/R/lib
```

Comment:
This rpath doesn't make sense; pyzmq should _not_ be looking at anything from an R install. We should check the build pipeline to understand why something from this directory is being picked up.